biblatex-iso690.tex

```diff
@@ -17,15 +17,11 @@
 \newcommand*{\biblatex}{\sty{biblatex}\xspace}
 
 \usepackage[
-  backend=biber,
   style=iso-authoryear,
-  autolang=other,
+  sortlocale=cs_CZ,% We need this for proper sorting,
```

Comment:
My comment https://github.com/michal-h21/biblatex-iso690/pull/74#discussion_r362390949 was only about `bibencoding` (even though GitHub highlighted several lines).

Since the document is in English the `sortlocale=cs_CZ,` is indeed needed for Czech sorting.

You may or may not want to retain `autolang=other,` depending on whether or not it is actually needed.

Reply:
I removed other options:
* `backend=biber` because it is default by `biblatex`,
* `autolang=other` because it is included in `iso.bbx` as a default option,
* `spacecolon=false` because it is included in `iso.bbx` as a default option.

Reply:
Ah, makes sense.  I didn't check the default `iso.bbx` options. (I actually like `backend=biber,` and always keep it to remind people to run Biber, but it's been the default for a long time.)